jablib/src/main/java/org/jabref/logic/net/ssl/TrustStoreManager.java

```diff
@@ -231,7 +231,7 @@ public void checkClientTrusted(X509Certificate[] chain, String authType) throws
 
     private static void setSystemTrustManager(X509TrustManager mergedTrustManager)
             throws NoSuchAlgorithmException, KeyManagementException {
-        SSLContext sslContext = SSLContext.getInstance("TLS");
+        SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

Comment:
Only "TLS" seems to fetch the old version. TLSv1.2 and v1.3 should be available. Not sure if we should use 1.2 and 1.3